src/state/slices/opportunitiesSlice/resolvers/idle/index.ts

```diff
@@ -30,6 +35,11 @@ export const idleStakingOpportunitiesMetadataResolver = async ({
 }: OpportunitiesMetadataResolverInput): Promise<{
   data: GetOpportunityMetadataOutput
 }> => {
+  const { getState } = reduxApi
+  const state: any = getState() // ReduxState causes circular dependency
+
+  const { IdleFinance } = selectFeatureFlags(state)
```

Comment:
As we've found out the hard way, upstream changes and/or outages might bork things, so having the ability to flag things off is always useful, even though the feature is now stable and the flag was effectively "useless"